deploy-config/deploy-config-holesky.toml

```diff
@@ -25,17 +26,17 @@ min_withdrawals_batch_size = 1
 
 [dual_governance_config_provider]
 first_seal_rage_quit_support = 300
+second_seal_rage_quit_support = 1500
 min_assets_lock_duration = 300
 rage_quit_eth_withdrawals_delay_growth = 1296000
```

Comment:
The delay growth looks enormous for the current max and min values. Reconsider these values.